gdk/commands/component/list.py

```diff
@@ -35,7 +35,7 @@ def get_component_list_from_github(url):
         comp_list = comp_list_response.json()
         return comp_list
     except Exception as e:
-        logging.error(e)
+        logging.error(e, exc_info=True)
```

Comment:
I think it may be redundant to use (e, exc_info)

Reply:
That's right but in other places where we just log with `e`,  we raise meaningful custom exception messages. But, here, we just proceed even when the exception occurs.  

So, this exc_info helps us log when and where details of it in the cases where logs with just `e` produces incomplete messages like this `TypeError: 'str' object is not callable`